test-case/manual/check-pulseaudio-automatic-source-switch.md

```diff
@@ -0,0 +1,68 @@
+# Pulseaudio test - Automatic source switch
+Ouput and input sources should be automatic changed once an external device is connected or disconnected
+
+# Preconditions
+1. PulseAudio should be enabled
+2. UCM for the platform/device should be installed
+
+# Test Description
+* Output source and intput source should be automatic changed once connect or disconnect headset
+* Output source should be automatic changed once connect or disconnect HDMI
+
+## Headset Test Case:
+1. Boot up system without HDMI and headset connection
+2. `parecord 1.wav -vvvv`, build-in device, such as DMIC is used for capturing. Say something then ctrl+c to stop
+3. `paplay 1.wav -vvvv`, internal speaker is used for playback. Audio is output successfully then press ctrl+c to stop
+4. Wait for about 3s till runtime PM is suspended, can check via `cat "/sys/bus/pci/devices/0000:00:1f.3/power/runtime_status"`
```

Comment:
you need to point this pci id `00:1f.3` is for intel audio device from `lspci`

Reply:
updated: 4. Wait for about 3s till runtime PM is suspended, can check via `cat "/sys/bus/pci/devices/0000:00:1f.3/power/runtime_status"`, the pci device id can be find from `lspci` for Multimedia audio controller